pkg/auth/dependency/sso/azureadv2.go

```diff
@@ -32,7 +28,46 @@ type azureadv2OpenIDConfiguration struct {
 
 func (f *Azureadv2Impl) getOpenIDConfiguration() (c azureadv2OpenIDConfiguration, err error) {
 	// TODO(sso): Cache OpenID configuration
-	endpoint := fmt.Sprintf(azureadv2ConfigurationURLFormat, f.ProviderConfig.Tenant)
+
+	tenant := f.ProviderConfig.Tenant
+	var endpoint string
+	// GUIDE(sso): Azure special tenant
```

Comment:
I don't think we should include guides in comment. It's better to put it in documentation/spec.

Reply:
Later on when we have guide, I will move this comment there.